tests/integration/test_charm.py

```diff
@@ -38,14 +38,21 @@ async def test_build_and_deploy(ops_test):
     assert all([endpoint.name == "grpc" for endpoint in relation.endpoints])
 
 
+@pytest.mark.abort_on_fail
 async def test_deploy_with_prometheus_and_grafana(ops_test):
     scrape_config = {"scrape_interval": "30s"}
-    await ops_test.model.deploy(PROMETHEUS, channel="latest/beta")
-    await ops_test.model.deploy(GRAFANA, channel="latest/beta")
-    await ops_test.model.deploy(PROMETHEUS_SCRAPE, channel="latest/beta", config=scrape_config)
+    await ops_test.model.deploy(PROMETHEUS, channel="latest/beta", trust=True)
+    await ops_test.model.deploy(GRAFANA, channel="latest/beta", trust=True)
+    await ops_test.model.deploy(
+        PROMETHEUS_SCRAPE, channel="latest/beta", trust=True, config=scrape_config
+    )
```

Comment:
Let's use `latest/stable` like in other charms.

Reply:
Pushed a commit to use `latest/edge` channel for mlmd and `latest/stable` channel for other charms.